ramls/oaipmh/oaipmhinstance.json

```diff
@@ -0,0 +1,172 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Oai pmh view response schema",
+  "properties": {
+    "items": {
+      "type": "array",
+      "description": "item fields",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "itemId",
+              "type": "string"
+            },
+            "volume": {
+              "description": "Item volume",
+              "type": "string"
+            },
+            "location": {
+              "description": "Item effective location",
+              "type": "object",
+              "properties": {
+                "location": {
```

Comment:
`items.location.location`? Can we do something better here? We only need the location name, not all these other ID, why do we need them here? It makes it difficult to determine what is needed by the edge/mod-rtac contract. I am having a hard time relating these properties to what mod-rtac is returning for the batch response. Where is the JSON used?

Reply:
Since both RTAC and OAI-PMH requst the same API in mod-inventory-storage, this schema is the schema of the response of that API. I think the confusion was introduced while copypasting. 
I'll leave only those fields that are used by RTAC.